resources/user-agents/browsers/seamonkey/seamonkey-2-30-on.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "SeaMonkey #MAJORVER#.#MINORVER#",
-  "versions": ["2.42", "2.41", "2.40", "2.39", "2.38", "2.36", "2.35", "2.34", "2.33", "2.32", "2.31", "2.30"],
+  "versions": ["2.46", "2.42", "2.41", "2.40", "2.39", "2.38", "2.36", "2.35", "2.34", "2.33", "2.32", "2.31", "2.30"],
```

Comment:
2.46 is the latest release of SeaMonkey. I didn't see anything between 2.46 and 2.40 listed on their download page, so I didn't add anything else.

http://www.seamonkey-project.org/releases/#old_unofficial